### PR TITLE
chore(project): add `object-curly-newline` eslint rule

### LINKED ISSUE #966: [refactor]: add object-curly-newline rule in eslint
To keep the code style uniform, we need this rule

rule details: https://eslint.org/docs/rules/object-curly-newline

You should add this rule to `eslintCommon -> base -> rules` in [.eslint.common.js](https://github.com/netless-io/flat/blob/ef6668bbcb75503b0fb6c502601ec2de910b2df8/.eslint.common.js). eg:

https://github.com/netless-io/flat/blob/ef6668bbcb75503b0fb6c502601ec2de910b2df8/.eslint.common.js#L133-L134

just like this:

图片

### PR DESCRIPTION
**Pull request by syt-honey**
1. add `object-curly-newline` rule
2.  turn off `comma-dangle` rule, because it conflicts with `prettier trailingComma`
3. format project code

closed: #966

### CODE REVIEW

File: desktop/main-app/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "ship:mac": "yarn build && yarn pack:mac",
     "postinstall": "yarn _download:agora-addon",
     "lint": "lint-staged",
+    "fix": "eslint --fix 'src/**/*.{ts, tsx}' 'typings/*.ts'",
```
Comment (BlackHole1) [inline desktop/main-app/package.json]: remove

File: desktop/renderer-app/package.json [hunk 2]
```diff
@@ -112,6 +112,7 @@
     "start": "cross-env NODE_ENV=development node scripts/start.js",
     "build": "cross-env NODE_ENV=production webpack --config webpack/webpack.prod.js",
     "build:analyzer": "cross-env NODE_ENV=production ANALYZER=true webpack --config webpack/webpack.prod.js",
-    "lint": "lint-staged"
+    "lint": "lint-staged",
+    "fix": "eslint --fix 'src/**/*.{ts, tsx}' 'typings/*.ts'"
```
Comment (BlackHole1) [inline desktop/renderer-app/package.json]: remove

File: packages/flat-components/package.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
   },
   "scripts": {
     "lint": "lint-staged",
+    "fix": "eslint --fix 'src/**/*.{ts, tsx}' 'typings/*.ts'",
```
Comment (BlackHole1) [inline packages/flat-components/package.json]: remove

File: web/flat-web/package.json [hunk 4]
```diff
@@ -85,7 +85,8 @@
     "start": "vite --open",
     "build": "cross-env NODE_OPTIONS=\"--max-old-space-size=6144\" vite build",
     "serve": "vite preview",
-    "lint": "lint-staged"
+    "lint": "lint-staged",
+    "fix": "eslint --fix 'src/**/*.{ts, tsx}' 'typings/*.ts'"
```
Comment (BlackHole1) [inline web/flat-web/package.json]: remove

File: yarn.lock [hunk 5]
```diff
@@ -1111,6 +1111,11 @@
   dependencies:
```
Comment (BlackHole1) [inline yarn.lock]: reset `yarn.lock`

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Comment by BlackHole1** [comment]
we don't need `fix script`.

you should reset `yarn.lock` file change, because `package.json` no any update

**Comment by syt-honey** [comment]
> we don't need `fix script`.
> 
> you should reset `yarn.lock` file change, because `package.json` no any update

done.

**Comment by BlackHole1** [comment]
@syt-honey CI has failed, can you look at it?

**Comment by BlackHole1** [comment]
`object-curly-newline` unable to meet demand

**Pull request closed without merging**